Experimental/DistributedAuthoritySample/Assets/Scripts/Player/CameraControl.cs

```diff
@@ -0,0 +1,127 @@
+using System.Collections;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using Unity.Cinemachine;
+using Unity.Netcode;
+
+namespace Unity.Multiplayer.Samples.SocialHub.Player
+{
+    public class CameraControl : MonoBehaviour
+    {
+        public CinemachineCamera freeLookVCam;
```

Comment:
These 4 public fields can be private and set via the editor using SerializeField property.